www/src/components/homepage/homepage-features.js

```diff
@@ -15,6 +16,10 @@ const HomepageFeatures = () => (
       flex: `0 1 auto`,
       flexWrap: `wrap`,
       px: 8,
+      pb: 8,
```

Comment:
Any reason to not use theme-ui's array syntax for `pb` here?

Reply:
This is my first PR on Gatsby, so basically I am not aware of that syntax, if you feel doing it with the array syntax is much cleaner way, then I can read about it and update the code, please let me know what you feel?

Reply:
I have used the theme-ui's array syntax for padding-bottom

Reply:
Hey Abhishek! AFK currently, gotta check, but the first position in the `theme-ui` array syntax should result in a "mobile-first" CSS rule without any `media` … hope that makes sense.

Reply:
So the `pb: [8, 0]` should produce something like

```
.element { padding-bottom: 2.5rem } // that's the 8

@media screen and (min-width: 400px)
.element {
    padding-bottom: 0; // that's the 0
}

Reply:
@fk I have exactly same thing which you described above, everything is working perfect.

The 400px and 550px thing which I described above is not related to this PR.

Reply:
Ahh sweet! So let's get this in! <3